internal/api/template_config_service.go

```diff
@@ -0,0 +1,89 @@
+package api
+
+import (
+	"context"
+
+	"github.com/DIMO-Network/vehicle-signal-decoding/internal/core/commands"
+	"github.com/DIMO-Network/vehicle-signal-decoding/internal/core/queries"
+
+	db "github.com/DIMO-Network/shared/db"
+	grpc "github.com/DIMO-Network/vehicle-signal-decoding/pkg/grpc"
+	"github.com/rs/zerolog"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+type TemplateConfigService struct {
+	grpc.TemplateConfigServiceServer
+	logger *zerolog.Logger
+	DBS    func() *db.ReaderWriter
+}
+
+func NewTemplateConfigService(logger *zerolog.Logger, dbs func() *db.ReaderWriter) grpc.TemplateConfigServiceServer {
+	return &TemplateConfigService{logger: logger, DBS: dbs}
+}
+
+func (s *TemplateConfigService) CreateTemplate(ctx context.Context, in *grpc.UpdateTemplateRequest) (*emptypb.Empty, error) {
+	service := commands.NewCreateTemplateCommandHandler(s.DBS)
+	_, err := service.Execute(ctx, &commands.CreateTemplateCommandRequest{
+		Name:               in.GetTemplate().GetName(),
```

Comment:
normally the `Get` methods shouldn't be needed, you should be able to just use the property directly, eg. `Template.Name`

Reply:
done